packages/components/inputs/select-utils/src/clear-indicator/clear-indicator.tsx

```diff
@@ -37,6 +37,10 @@ const ClearIndicator = (props: TClearIndicator) => {
       style={getStyles('clearIndicator', props) as CSSProperties}
       title={intl.formatMessage(messages.clearButtonLabel)}
       aria-label={intl.formatMessage(messages.clearButtonLabel)}
+      // overriding the default `aria-hidden` prop value to make the component accessible by keyboard - https://github.com/JedWatson/react-select/issues/4793
+      aria-hidden={false}
+      // only onMouseDown and onTouchEnd event handlers are passed by `react-select` to the component by default, which makes it not accessible by keyboard
+      onClick={onMouseDown}
```

Comment:
Another issue I noticed is that by default the `ClearIndicator` does not trigger any action on “Enter” key stroke. Therefore, the component is not accessible by keyboard when `isClearable` prop is passed to the component. The reason is that by default only `onMouseDown` and `onTouchEnd` event handlers are passed to the component. I am suggesting using the `onClick` instead, but perhaps for some reason this is not what we want

Reply:
For me, it also makes sense to use the `onClick` handler.